modules/shared/components/organisms/PostCreation/PostCreation.tsx

```diff
@@ -159,6 +159,7 @@ const PostCreation: FC<IPostCreation.IProps> = ({
         await logoutUser();
         redirectToLoginPage();
       }
+      window.location.reload();
```

Comment:
Wohhoo wohhoo.. that's too aggressive for SPA app 😂

Reply:
There is another way using next router would that be better?